rasa/core/channels/facebook.py

```diff
@@ -88,7 +89,7 @@ def _is_quick_reply_message(message: Dict[Text, Any]) -> bool:
         else:
             logger.warning(
                 "Received a message from facebook that we can not "
```

Comment:
I don't think there is an established style for this yet, couldn't find anything that's explicitly about f-strings, but indentation in brackets etc is supposed to line up, so it might be better to add `f` to every line

Reply:
Anyway, I don't want to go changing it everywhere now. Let's hope black do something about this at some point. For now I think we should stick with changing it whenever we encounter it in code but not go after it as it doesn't matter much